instance/tasks.py

```diff
@@ -66,6 +66,7 @@ def watch_pr():
             instance, created = OpenEdXInstance.objects.get_or_create(
                 sub_domain=pr_sub_domain,
                 fork_name=pr.fork_name,
+                github_pr_number=pr.number,
```

Comment:
@antoviaque Since the PR number is also included in the subdomain, which is one of the keys in the `get_or_create()` call, it would actually be possible to have multiple PRs for the same branch. (I haven't ever found this useful in practice, but who knows.)